app/models/manageiq/providers/nutanix/infra_manager/vm/operations/power.rb

```diff
@@ -0,0 +1,71 @@
+module ManageIQ::Providers::Nutanix::InfraManager::Vm::Operations::Power
+  extend ActiveSupport::Concern
+
+  # included do
+  #   # Existing shelve operations
+  #   supports :start
+  #   supports :shelve do
+  #     if %w[on off suspended paused].exclude?(current_state)
+  #       _("The VM can't be shelved, current state has to be powered on, off, suspended or paused")
+  #     end
+  #   end
+
+  #   supports :shelve_offload do
+  #     if current_state != "shelved"
+  #       _("The VM can't be shelved offload, current state has to be shelved")
+  #     end
+  #   end
+  # end
+
+
+  def start
+    raw_start
+  rescue => err
+    raise MiqException::MiqVmError, _("Start operation failed: %{message}") % {:message => err.message}
+  end
+
+  def raw_start
+    with_provider_connection do |connection|
+      api = ::NutanixVmm::VmApi.new(connection)
+
+      _, _, headers = api.get_vm_by_id_0_with_http_info(ems_ref)
+      etag = headers['etag'] || headers['ETag']
+      raise "ETag missing from VM GET response" if etag.nil?
+
+      request_id = SecureRandom.uuid
+      api.power_on_vm_0(ems_ref, etag, request_id)
+    end
+
+    update!(:raw_power_state => "OFF")
+  end
+
+  def raw_stop
+    with_provider_connection do |connection|
+      api = ::NutanixVmm::VmApi.new(connection)
+
+      _, _, headers = api.get_vm_by_id_0_with_http_info(ems_ref)
+      etag = headers['etag'] || headers['ETag']
+      raise "ETag missing from VM GET response" if etag.nil?
+
+      request_id = SecureRandom.uuid
+      api.power_off_vm_0(ems_ref, etag, request_id)
+    end
+
+    update!(:raw_power_state => "ON")
+  end
+
+  def raw_pause
+    raise NotImplementedError, _("Pause operation is not supported for Nutanix VMs")
+  end
+
+  def raw_suspend
+    raise NotImplementedError, _("Suspend operation is not supported for Nutanix VMs")
+  end
```

Comment:
Don't need to define these raising that this isn't implemented because that is what we use supports feature for.
As long as the VM responds false to `vm.supports?(:pause)` then we're good